host/CMakeLists.txt

```diff
@@ -248,12 +248,18 @@ if (OE_SGX)
     ../common/sgx/report.c
     ../common/sgx/collateral.c
     ../common/sgx/sgxcertextensions.c
+    ../common/sgx/sgxmeasure.c
     ../common/sgx/tcbinfo.c
     ../common/sgx/tlsverifier.c
     ../common/sgx/verifier.c
+    ../common/sgx/eeid_verifier.c
```

Comment:
Should this only be included if eeid flag is enabled?

Reply:
Yes, good catch, thanks!